src/context_store/orchestrator.py

```
@@ -338,7 +342,15 @@ async def start_lifecycle(self) -> None:
 
     async def dispose(self) -> None:
         """全アダプターのリソースを解放する。"""
-        await self._lifecycle_manager.graceful_shutdown()
+        # まずライフサイクルマネージャーを終了させ、タスクの完了を待機する
+        try:
+            await self._lifecycle_manager.graceful_shutdown()
+        except Exception as exc:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)